generatedocs/data/config/scheduler/description.md

```diff
@@ -1,3 +1,17 @@
 ## Scheduler section
 
 This section of the JSON file contains scheduler settings for the users in the load scenario.
+
+### Using "reconnectsettings"
+
+If `reconnectsettings.reconnect` is enabled the tool will attempt the following:
+
+* Re-connect the websocket.
+* Get the currently opened app in re-attached engine session.
+* Re-subscribe to the same object has before getting disconnected.
+* Restart the action which was ongoing during the disconnect.
```

Comment:
Suggestion:
"* Restart the action that was ongoing when the disconnection occurred."